src/vm/gdbjit.cpp

```diff
@@ -2178,7 +2201,7 @@ bool NotifyGdb::CollectCalledMethods(CalledMethod* pCalledMethods)
         pList = pList->GetNext();
         delete ptr;
     }
-
+    SymbolCount = i;
```

Comment:
I don't understand the reason for this change. The loop will always exit when `i` is equal to the `SymbolCount`, so this assignment doesn't make sense. It seems the previous version of the code would have the same effect.